examples/contour/README.md

```diff
@@ -20,10 +20,9 @@ This configuration has several advantages:
 For detailed instructions on how to configure the required certs manually, see the [step-by-step TLS HOWTO](../../docs/grpc-tls-howto.md).
 
 ## Deploy Contour
-
 Either:
 
-1. Run `kubectl apply -f https://projectcontour.io/quickstart/contour.yaml`
+1. Run `kubectl apply -f https://raw.githubusercontent.com/projectcontour/contour/master/examples/render/contour.yaml
```

Comment:
The quickstart points to the latest stable release, today that is v0.15. There are some inconsistencies that come up due to this which we are discussing in #1619. 

TLDR: Let's keep this pointing to the quickstart which gets updated each new release of Contour.

Reply:
Should we put a note or something specifying the updated contour.yml if doing a 1.0.0-1beta install until the work in #1619 is completed. As currently even following the docs and links directly from projectcontour.io leads to this same page?

example flow we followed:
https://projectcontour.io/ 
  --> Documentation 
     --> v1.0.0-beta.1 (https://github.com/heptio/contour/tree/v1.0.0-beta.1/docs) 
         --> README.md Deployment Options (https://github.com/projectcontour/contour/blob/v1.0.0-beta.1/docs/deploy-options.md) 
    --> Recommended Installation details Link in paragraph contour example 
          (https://github.com/projectcontour/contour/blob/v1.0.0-beta.1/examples/contour/README.md) 
        --> leads to quickstart link still showing old deployment but all wording on page is updated to call out new deployment method (IE daemon sets for envoy containers).

Reply:
I agree this is broken. The issue is there are multiple links that all use this quickstart. We'll look to fix that in #1619 which should happen very soon so that this is a much smoother experience.

Reply:
doc changed as requested to point back to quickstart